<?xml version="1.0"?>
<launch>
    <!-- <arg name="config_path" default="$(find multi_localizer)/config" /> -->
    <arg name="roomba"      default="roomba1" />

    <!-- object_data_publisher -->
    <node pkg="dom_estimator" type="object_data_publisher" name="$(arg roomba)_object_data_publisher" >
        <!-- remap -->
		<remap from="pose_in"  to="$(arg roomba)/amcl_pose" />
        <remap from="odom_in"  to="$(arg roomba)/roomba/odometry" />
		<remap from="obj_in"   to="$(arg roomba)/object_positions" />
		<remap from="pose_out" to="$(arg roomba)/pose" />
		<remap from="obj_in"   to="$(arg roomba)/objects_data" />

        <!-- params -->
		<param name="ROBOT_NAME"         value="$(arg roomba)" />
		<param name="MAP_FRAME_ID"       value="map" />
        <param name="BASE_LINK_FRAME_ID" value="$(arg roomba)/base_link" />
        <!-- <param name="LASER_FRAME_ID"     value="$(arg roomba)/laser" /> -->
        <!-- <param name="CAMERA_FRAME_ID"    value="$(arg roomba)/camera" /> -->

        <!-- rosparams -->
        <!-- <rosparam command="load" file="$(arg config_path)/params/tf_broadcaster.yaml" /> -->
    </node>

</launch>